input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
 Parent: AUBaseDiagnosticRequest
-Id: au-erequesting-servicerequest
-Title: "AU eRequesting ServiceRequest"
+Id: au-erequesting-diagnosticrequest
+Title: "AU eRequesting Diagnostic Request"
```

Comment:
Commenting here to get an inline comment. 

Please update CapabilityStatements - all capability statements and the source file for the Summary of Must Support and References Between Profiles will need to be updated.

Reply:
Updated CapStats + resourcereferencesummary file.